drivers/column.go

```diff
@@ -11,6 +11,7 @@ import (
 type Column struct {
 	Name      string `json:"name" toml:"name"`
 	Type      string `json:"type" toml:"type"`
+	NullType  string `json:"null_type" toml:"null_type"`
```

Comment:
What's the purpose of this given `Nullable` + `Type`?

Reply:
no purpose - just an unintentional artifact of the previous method of using the all-null struct, and this was not cleaned up completely.